ci/build_python.sh

```diff
@@ -22,7 +22,7 @@ conda config --set path_conflict prevent
 
 sccache --zero-stats
 
-RAPIDS_PACKAGE_VERSION=$(head -1 ./VERSION) rapids-conda-retry mambabuild \
+RAPIDS_PACKAGE_VERSION=$(rapids-generate-version) rapids-conda-retry mambabuild \
```

Comment:
Note that the function is being called once above, hence why I previously just read the output.

Reply:
Here is the line Vyas referenced

https://github.com/rapidsai/kvikio/blob/b00141379b8ec777b0f3081069986c70552f880e/ci/build_python.sh#L16

And we need to that line as the `VERSION` file has to be written out there

Reply:
I think this is slightly preferably. I misunderstood the original code, I though that `./VERSION` was written just to be read here :)

Reply:
Added a comment to that line. Also shared a few examples below